source/App.cpp

```diff
@@ -1381,26 +1238,17 @@ void App::onUserInput(UserInput* ui) {
 	for (GKey dummyShoot : keyMap.map["dummyShoot"]) {
 		if (ui->keyPressed(dummyShoot) && (sess->presentationState == PresentationState::feedback)) {
 			Array<shared_ptr<Entity>> dontHit;
-			dontHit.append(m_currentMissDecals);
+			//dontHit.append(m_currentMissDecals);
 			dontHit.append(m_explosions);
-			for (auto projectile : m_projectileArray) { dontHit.append(projectile.entity); }
+			//for (auto projectile : m_projectileArray) { dontHit.append(projectile.entity); }
```

Comment:
again we removed a bunch of stuff from the `dontHit` list, seems like this might cause collisions with miss decals and projectiles that we didn't want.

Reply:
See the [comment above](https://github.com/NVlabs/abstract-fps/pull/114#discussion_r403176424) for why we don't add other things here.

We should verify that there aren't issues with miss decals/multiple projectiles in flight simultaneously though.